packages/subgraph/.gitignore

```diff
@@ -0,0 +1,6 @@
+build/
+generated/
+abis/*
+yarn.lock
```

Comment:
We should probably have lockfiles in the repo, I don't think it's a good idea to `.gitignore` them.

Reply:
Resolved